verify/descriptiveness.go

```diff
@@ -36,16 +36,15 @@ Someone reading the PR description without clicking any issue links should be ab
 }
 
 // checkPRDescriptiveness
-func checkPRDescriptiveness(requiredLines int) action.ValidateFunc {
+func checkPRDescriptiveness(requiredCharacters int) action.ValidateFunc {
 	return func(pr *github.PullRequest) (string, string, error) {
-		lineCnt := 0
-		for _, line := range strings.Split(pr.GetBody(), "\n") {
-			if strings.TrimSpace(line) == "" {
-				continue
+		for _, label := range pr.Labels {
+			if *label.Name == skipDescriptivenessCheckLabel {
+				return "Skipping descriptiveness check!", "", nil
 			}
-			lineCnt++
 		}
-		if lineCnt < requiredLines {
+
+		if len(pr.GetBody()) < requiredCharacters {
```

Comment:
```suggestion
		if len(strings.TrimSpace(pr.GetBody())) < requiredCharacters {
```

Reply:
Guess this wil prevent adding spaces at the end of the description to get to the 80 chars.